README.md

```diff
@@ -42,6 +42,24 @@ More on web renderers here: https://flutter.dev/docs/development/tools/web-rende
           webRenderer: canvaskit
 ```
 
+The possible values for webRenderer are:
+
+- auto: Use the default Flutter setting, which is based on the device and browser capabilities.
+- canvaskit: Use the CanvasKit renderer, which uses WebGL and WebAssembly to render graphics.
+- html: Use the HTML renderer, which uses DOM elements and CSS to render graphics.
+- both: Use both renderers and generate two versions of the web app, one in the root folder (here would be the CanvasKit version) and one in a subfolder specified by the htmlRoute property.
```

Comment:
In which cases would you want to use both? It feels like it's maybe a bit too specific to have in the action

Reply:
I agree, then the user could use two instances of the action